File: BLL/Infrastructure/Models/UserProfileModel.cs
using Common.Enums;

namespace BLL.Infrastructure.Models
{
    public class UserProfileModel
    {
        public int UserId { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public byte[] ProfilePicture { get; set; }

        public string Address { get; set; }

        public string PhoneNumber { get; set; }

        public DateTime BirthDate { get; set; }

        public string Email { get; set; }

        public string Login { get; set; }

        public DateTime RegistrationDate { get; set; }

        public Role Role { get; set; }
    }
}

File: Frontend/BibliWorm/src/app/core/models/user/user-profile-model.ts
import { Role } from "../../enums/role";

export interface UserProfileModel {
    userId: number,
    firstName: string,
    lastName: string,
    profilePicture: string,
    address: string,
    phoneNumber: string,
    birthDate: Date,
    birthDateString: string,
    email: string,
    login: string,
    registrationDate: Date,
    role: Role
}
